README.md

```diff
@@ -69,6 +69,21 @@ The script will bypass all prompts if the values are provided by environment var
 |clientSecret|From json blob|
 |registryKey| From json blob, Base64 encoded docker login credentials|
 
+# Environment File for Database Accounts
```

Comment:
You should note in these instructions that since we're using systemctl to manage the services, this needs to be in the predefined environment variable file and which service leverages it. In short, make sure to add that this environment variable needs to be defined in the `/etc/default/cyral-authenticator` file.